pt-br/NuzlockeTools.js

```diff
@@ -0,0 +1,42 @@
+export default {
+  'Sections': {
+    'DeathTracking': {
+      'Title': 'Rastreio de Mortes',
+      'DeathExplanation': 'Se um Pokémon for revivido e chegar a 0 de HP de novo, será contabilizado como 2 mortes. Clicar na cruz resetará as mortes para sempre.',
+      'NoResults': 'Sem mortes que condizem com sua pesquisa',
+      'NoDeaths': 'Você não teve nenhuma morte nesta sessão',
+      'TimeOfDeath': 'Hora da Morte',
+      'ResetAllDeaths': 'Resetar todas as mortes',
+      'ResetConfirmation': {
+        'Title': 'Tem certeza?',
+        'Body': 'Remover todas as suas mortes é irreversível, tem certeza que deseja resetar todas as mortes rastreadas?',
+        'Confirm': 'Sim, Resete minhas mortes!'
+      },
+      'ResetSuccess': {
+        'Title': 'Reset realizado com sucesso',
+        'Body': 'Suas mortes foram deletadas'
+      }
+    },
+    'ShowdownExport': {
+      'Title': 'Exportação para Showdown',
+      'ContentTitle': 'Exporte seu time para Showdown',
+      'ProcessDescription': 'Pode ser usado para importar seu time para o Showdown e até mesmo diversas calculadoras de dano online.',
+      'NoShowdownData': 'Houve um erro ao exportar seu time. Reporte isso no Discord do Pokélink.'
+    },
+    'RouteTracking': {
+      'Title': 'Rastrador de Rotas',
+      'SupportInfo': 'Rotas agora são rastreadas automaticamente nas Gerações 3,4,5,6 e 7 e também em sessões manuais utilizando o campo "Local de encontro".',
```

Comment:
for context - the game uses the term "location met" as in "the location a pokemon was met" (encountered). If you're happy with using the term "meeting place", then that's fine. As long as the wider portuguese pokemon community uses the same wording?

Reply:
'Local Encontrado' should work as well, maybe better, both people in the community shouldn't have trouble understanding